aperture.go

```diff
@@ -189,11 +189,11 @@ func setupLogging(cfg *config) error {
 // getTLSConfig returns a TLS configuration for either a self-signed certificate
 // or one obtained through Let's Encrypt.
 func getTLSConfig(serverName string, autoCert bool) (*tls.Config, error) {
-	var tlsKeyFile, tlsCertFile string
-	switch {
-	// When using autocert, we set a TLSConfig on the server so the key and
-	// cert file we pass in are ignored and don't need to exist.
-	case autoCert:
+	// If the --autocert flag is set, use the autocert library that will
```

Comment:
It feels a bit strange to mention the exact flag name here.

Reply:
Agree.